packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-load/main/go/integrationsteps.md

```diff
@@ -23,6 +23,7 @@ func (s *Server) LoginHandler(w http.ResponseWriter, r *http.Request) {
     BaseUrl           string
     ClientId          string
     Issuer            string
+    Otp                string
     State             string
     Nonce             string
     InteractionHandle string
```

Comment:
Missing the OTP in data variable initialization (data := customData{)

Reply:
fixed